README.md

```diff
@@ -214,18 +268,20 @@ nv-ipam accepts the following CNI configuration:
 {
     "type": "nv-ipam",
     "poolName": "my-pool",
-    "kubeconfig": "/etc/cni/net.d/nv-ipam.d/nv-ipam.kubeconfig",
-    "dataDir": "/var/lib/cni/nv-ipam",
+    "daemonSocket": "unix:///var/lib/cni/nv-ipam/daemon-socket",
+    "daemonCallTimeout": 5,
```

Comment:
renamed to daemonCallTimeoutSeconds

Reply:
is there any call that would take more than a second ?
do we really want to expose this option ? i cant imagine why a user may want to change the value from whatever default we will have.

TBH if an RPC call takes more than a second something is either really wrong with the system or our code.

Reply:
This can help us to survive on a very large scale when we have many pools and lot of allocations in them (thousands). Currently we are using single file with serialized access. If we have many concurrent requests and file write time is quite long (because of the large file size or because of the slow IO) we can be in trouble with small default timeout.

Reply:
ack, when we reach this point we will need to change the store :)